Feature/docker (#496)
PennyLaneAI/pennylane-lightning: docker/README.md

# PennyLane and the Lightning plugins

PennyLane is an open-source software framework for quantum machine learning, quantum chemistry, and quantum computing, with the ability to run on all hardware. Maintained with ❤️ by Xanadu.

## Documentation

> For more information on PennyLane, including the demos, APIs and development guide, visit the [PennyLane documentation site](https://pennylane.ai/).

Reviewer: It would be great if we had direct links to Lightning and Lightning-GPU docs.

# PennyLane and the Lightning plugins

PennyLane is an open-source software framework for quantum machine learning, quantum chemistry, and quantum computing, with the ability to run on all hardware. Maintained with ❤️ by Xanadu.

## Documentation

For more information on PennyLane, including the demos, APIs and development guide, visit the [PennyLane documentation site](https://pennylane.ai/).

> Listed below are links to the Lightning-plugins documentation:
> 
> - [pennylane-lightning doc](https://docs.pennylane.ai/projects/lightning/en/stable/)
> - [pennylane-lightning-gpu doc](https://docs.pennylane.ai/projects/lightning-gpu/en/stable/)
> - [pennylane-lightning-kokkos doc](https://docs.pennylane.ai/projects/lightning-kokkos/en/latest/)

Reviewer: ```suggestion
You can also check the Lightning-plugins documentation:

- [pennylane-lightning](https://docs.pennylane.ai/projects/lightning/en/stable/)
- [pennylane-lightning-gpu](https://docs.pennylane.ai/projects/lightning-gpu/en/stable/)
- [pennylane-lightning-kokkos](https://docs.pennylane.ai/projects/lightning-kokkos/en/latest/)
```
